<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGLFrame" name="glWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="toolTab">
       <attribute name="title">
        <string>Tools</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>77</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="GLSLEditor" name="editorTab">
       <attribute name="title">
        <string>GLSL Editor</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEnableEdit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEnableEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLFrame</class>
   <extends>QWidget</extends>
   <header>qglframe.h</header>
   <container>1</container>
   <slots>
    <signal>xRotationChanged(int)</signal>
    <signal>yRotationChanged(int)</signal>
    <signal>zRotationChanged(int)</signal>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>prevModel()</slot>
    <slot>nextModel()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>GLSLEditor</class>
   <extends>QWidget</extends>
   <header>glsleditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>prevButton</sender>
   <signal>pressed()</signal>
   <receiver>glWidget</receiver>
   <slot>prevModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>pressed()</signal>
   <receiver>glWidget</receiver>
   <slot>nextModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
